apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -58,6 +58,17 @@ class GameLabVisualizationColumn extends React.Component {
     }
   };
 
+  pickerTouchMove = e => {
+    if (this.props.pickingLocation) {
+      const touch = e.nativeEvent.touches[0];
+      const divGameLabRect = this.divGameLab.getBoundingClientRect();
+      this.props.updatePicker({
+        x: Math.floor(touch.clientX - divGameLabRect.left),
+        y: Math.floor(touch.clientY - divGameLabRect.top),
+      });
+    }
+  };
```

Comment:
I don't love that we have two totally separate code paths for mouse vs. touch events.  I'm worried this will open us up to bugs in the future, where we make fixes for one input type but miss the other code path.

Can we map the coordinates in each handler, then call into the same base code?

```js
pickerMove = (x, y) => {
  if (this.props.pickingLocation) {
    const divGameLabRect = this.divGameLab.getBoundingClientRect();
    this.props.updatePicker({
      x: x - divGameLabRect.left,
      y: y - divGameLabRect.top,
    });
  }
};

pickerMouseMove = e => {
  this.pickerMove(e.pageX, e.pageX);
};

pickerTouchMove = e => {
  this.pickerMove(e.touches[0].clientX, e.touches[0].clientY);
};
```